src/bootstrap.js

```diff
@@ -25,9 +25,12 @@ function bootstrap(opts = {}) {
   /**
    * Create mount element dynamically.
    */
-  const el = document.createElement('div');
-  el.id = 'nc-root';
-  document.body.appendChild(el);
+  const el = document.getElementById('nc-root');
+  if (!el) {
+    document.createElement('div');
```

Comment:
This needs to be assigned to `el`, so that you can append it below.

Reply:
Oops, I have that on mine, forgot to push 👍 @tech4him1 good catch